posydon/popsyn/normalized_pop_mass.py

```diff
@@ -107,30 +108,74 @@ def fraction_simulated(f_bin, m_1, m_2, m_min, m_max,
         alpha2 = 2.35
         alpha3 = 2.35
     else:
-        Pwarn("Scheme not included yet: primary_mass_scheme="
-              f"{kwargs['primary_mass_scheme']}, secondary_mass_scheme"
-              f"={kwargs['secondary_mass_scheme']}", "UnsupportedModelWarning")
-        return np.nan, np.nan, np.nan
-#        raise ValueError("Scheme not included yet")
+        raise ValueError("Scheme not included yet")
```

Comment:
I'd prefer to keep a warning here. Because there might be use cases where you don't care about the underlying mass for a different model. But raising an error, will make it impossible to use data without underlying mass, because the code stops here.